src/ide/completion/path.rs

```diff
@@ -34,22 +37,25 @@ pub fn path_suffix_completions<'db>(
     db: &'db AnalysisDatabase,
     ctx: &AnalysisContext<'db>,
     was_node_corrected: bool,
+    only_top_level_declarative_macros: bool,
```

Comment:
I'm not a big fan of this function right now, it has not one (which i already disliked before) but two boolean parameters

We should split this up somehow so it's more modular and compose 2 different functions from this (we use it in 2 places with `true` and `false` respectively)